pywwt/layers.py

```diff
@@ -0,0 +1,100 @@
+import re
+
+class ImageryLayers():
+
+    def __init__(self, layer_list):
+        self._layers = {}
+        self._spectrum = ['gamma', 'x', 'uv', 'visible', 'ir', 'micro', 'radio', 'other']
+        for band in self._spectrum:
+            self._layers[band] = {}
+        
+        self._list2dict(layer_list)
+
+    def _list2dict(self, og_list):
+        for layer in og_list:
+            ind = 0
+
+            if(re.search(r'(?i)gamma',layer) != None):
+                self._add2dict(self._layers, layer, ind)
+                continue # automatically advance to next iteration
+
+            ind += 1
```

Comment:
I am assuming that ``ind`` gives the position in the spectrum list of bandpasses? If so I wonder whether it would be cleaner if ``add2dict`` simply took the name of the bandpass instead of these indices?